From 482b7362dedc57d0c9783bd83084919ccaa79e77 Mon Sep 17 00:00:00 2001
From: Waldemar Brodkorb <wbx@openadk.org>
Date: Wed, 14 Aug 2024 19:43:44 +0200
Subject: [PATCH] Fix implicit declaration of function basename in gcc-14.x

From Alpine Linux.
https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/main/iproute2/basename.patch?ref_type=heads

But also reported Upstream via netdev mailinglist by Gentoo people.

Signed-off-by: Waldemar Brodkorb <wbx@openadk.org>
Upstream: https://marc.info/?l=linux-netdev&m=172278781323421&w=2
---
 rdma/rdma.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/rdma/rdma.c b/rdma/rdma.c
index 131c6b2a..1e7db382 100644
--- a/rdma/rdma.c
+++ b/rdma/rdma.c
@@ -8,6 +8,9 @@
 #include "version.h"
 #include "color.h"
 
+/* For basename(3) prototype */
+#include <libgen.h>
+
 /* Global utils flags */
 int json;
 
-- 
2.39.2

